tested/languages/java/templates/declaration.mako

```diff
@@ -45,7 +45,7 @@
     % elif basic == BasicNumericTypes.RATIONAL:
         double\
     % elif basic == BasicObjectTypes.MAP:
-        Map<Object, Object>\
+        Map${'<String, Object>'}\
```

Comment:
The keys of the map type do not have to be strings. I realise now this is the case in JavaScript (since we use `object`), but perhaps it's better to use a `Map` in JavaScript instead of limiting the other languages to string-only keys.

Reply:
This is a current limitation of TESTed:
```python
@dataclass
class ObjectType(WithFeatures, WithFunctions):
    type: ObjectTypes
    data: Dict[str, 'Expression']
```
But I will create an issue to add not only support Strings as key.

Reply:
Ah, I (forgot/didn't know) about this limitation. In that case there's no problem.